backend/emails/emails.js

```diff
@@ -1,67 +1,52 @@
+const { EMAIL_RECIPIENTS } = require('../models/constants')
+
 const DivisionEmails = {
     software: 'software-leads@watonomous.ca',
     mechanical: 'mechanical-leads@watonomous.ca',
     business: 'business-leads@watonomous.ca',
     electrical: 'electrical-leads@watonomous.ca',
 }
 
-const ExpiringFundingReminder = (issueDetails) => {
-    const { toUrl, summary, funding_allocation, funding_allocation_spent } =
-        issueDetails
+const currencyFormatter = new Intl.NumberFormat('en-CA', {
+    style: 'currency',
+    currency: 'CAD',
+})
 
-    let htmlComponent = `
-        <div>
-            <a href=${toUrl}>${summary}</a> is expiring this term! Don't forget to
-            make any last minute purchases! This award has a total Funding
-            Allocation of ${funding_allocation} and ${funding_allocation_spent}{' '}
-            has been spent so far.
-        </div>
-    `
+const getEmailToSection = (emailToList) => {
+    const emailTo = []
 
-    return {
-        toName: 'financegroup@watonomous.ca',
-        subject: `${summary} is expiring`,
-        htmlPart: htmlComponent,
+    if (emailToList.includes(EMAIL_RECIPIENTS.admin)) {
+        emailTo.push('jw4he@watonomous.ca')
```

Comment:
this is a temporary hack, would make a TODO as once we migrate to Ben's RBAC we should be ok